transport/src/main/java/io/netty/channel/AbstractChannelHandlerContext.java

```diff
@@ -872,12 +892,90 @@ private AbstractWriteTask(Recycler.Handle handle) {
             super(handle);
         }
 
+        private static int estimateSize(Object o, Map<Class<?>, Integer> classSizes) {
+            int answer = 8 + estimateSize(o.getClass(), classSizes, null);
+
+            if (o instanceof ByteBuf) {
```

Comment:
@Scottmitch we not support FileRegion with SslHandler ;) So like I said it should be fine like it is for now.